specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-04-01-preview/costmanagement.json

```diff
@@ -1796,6 +1796,786 @@
         }
       }
     },
+    "/{scope}/providers/Microsoft.CostManagement/views": {
+      "get": {
+        "tags": [
+          "Views"
+        ],
+        "operationId": "Views_List",
+        "description": "Lists all views at the given scope.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "SubscriptionViewList": {
+            "$ref": "./examples/View/ViewList/ViewListBySubscription.json"
+          },
+          "ResourceGroupViewList": {
+            "$ref": "./examples/View/ViewList/ViewListByResourceGroup.json"
+          },
+          "ManagementGroupViewList": {
+            "$ref": "./examples/View/ViewList/ViewListByManagementGroup.json"
+          },
+          "BillingAccountViewList": {
+            "$ref": "./examples/View/ViewList/ViewListByBillingAccount.json"
+          },
+          "DepartmentViewList": {
+            "$ref": "./examples/View/ViewList/ViewListByDepartment.json"
+          },
+          "EnrollmentAccountViewList": {
+            "$ref": "./examples/View/ViewList/ViewListByEnrollmentAccount.json"
+          },
+          "BillingProfileViewList": {
+            "$ref": "./examples/View/ViewList/ViewListByBillingProfile.json"
+          },
+          "InvoiceSectionViewList": {
+            "$ref": "./examples/View/ViewList/ViewListByInvoiceSection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeViewParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ViewListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.CostManagement/externalBillingAccounts/{externalBillingAccountId}/views":{
```

Comment:
Hao, do we support /providers/microsoft.costmanagement/externalbillingaccounts/{name}/providers/microsoft.costmanagement/views? I only ask because I would really prefer not to document external billing accounts and subscriptions differently than every other scope. That will triple the size of our docs and it's simply not worth it. If anything, we can make a comment in our scope doc.

Reply:
We can not have namespace of CostManagement doubled in your example above, and have to separately list two scopes with external billing account and external subscription